ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/internal/pojoadapter/CallResult.java

```diff
@@ -14,15 +14,16 @@
  */
 package org.hyperledger.besu.ethereum.api.graphql.internal.pojoadapter;
 
-import org.hyperledger.besu.util.bytes.BytesValue;
+import org.apache.tuweni.bytes.Bytes;
 
-@SuppressWarnings("unused") // reflected by GraphQL
+@SuppressWarnings("unused")
+// reflected by GraphQL
```

Comment:
Can this comment make it back onto the same line as the annotation, or the one above the annotation?

Reply:
fixed, thanks!